client/src/Views/ReportsPage/ReportsPage.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <div class="reports-container">
     <template class="reports-container__v-tabs">
-      <v-tabs fixed-tabs v-model="tabs">
-        <v-tab>Monthly Reports</v-tab>
+      <v-tabs fixed-tabs v-model="tabs" :class="tabBarColoring">
+        <v-tab class="thingasasdassd">Monthly Reports</v-tab>
```

Comment:
What's the deal with this class? Is a placeholder for future styling or something? 😄

Reply:
It was a test to see what would show up and I totally spaced it 😂  I'll fix it